.github/workflows/check_on_push.yaml

```diff
@@ -5,6 +5,17 @@ on:
   pull_request:
 
 jobs:
+  version-check:
```

Comment:
I think we may ignore adding the job if nothing depends on it

Reply:
Deleted job from `check_on_push`